Docs/landing/layouts/partials/footer.html

```diff
@@ -1,6 +1,6 @@
 <div class="footer">
   <div class="copyright">
-    <p>Copyright © 2015 <a class="smalltext" href="http://www.mongodb.com">MongoDB, Inc</a>.
+    <p>Copyright © 2020 <a class="smalltext" href="http://www.mongodb.com">MongoDB, Inc</a>.
```

Comment:
maybe use JS to calculate this date and avoid this task each year?

Reply:
I see that we already use js https://github.com/vincentkam/mongo-csharp-driver/blob/fb39143e30a085eafc55154f67bcbb64051ff15b/Docs/landing/static/404.html#L81 but not sure that it's expected address :)

Reply:
Done